gl3n/linalg.d

```diff
@@ -2228,6 +2228,57 @@ struct Quaternion(type) {
         assert(almost_equal(q2.z, -1.060660));
         assert(almost_equal(q2.w, 0.7071067f));
     }
+    
+    /// Returns the quaternion as a vec3 (axis / angle representation).
+    vec3 to_axis_angle() {
+        vec3 ret;
+        quat this_normalized = this.normalized();
+        real angle = 2 * acos(this_normalized.w);
+        real denominator = sqrt(1.0 - (this_normalized.w)^^2);
+
+        if ( denominator < 0.0001) // avoid divide by 0
```

Comment:
Code Style would be:

```
if(denominator < 0.0001) {
```

No spaces and `{` on same line (same for `else` below)